airflow/www/static/js/connection_form.js

```diff
@@ -140,6 +140,55 @@ $(document).ready(function () {
     }
   }
 
+  $.fn.serializeObject = function () {
+    const o = {};
+    const a = this.serializeArray();
+    $.each(a, function () {
+      if (this.name === 'csrf_token') {
+        // dont send csrf token
+      } else if (this.name === 'conn_id') {
+        o.connection_id = this.value;
+      } else if (this.value !== '') {
```

Comment:
```suggestion
      } else if (this.value !== '' && this.name !== 'csrf_token') {
```

We shouldn't have an if statement that does nothing.